test/features/mobileapps.js

```diff
@@ -18,6 +18,7 @@ describe('Mobile Content Service', () => {
         })
         .then((res) => {
             assert.deepEqual(res.status, 200);
+            assert.deepEqual(res.headers.deprecation, 'Sat, 01 Jul 2023 00:00:00 GMT');
```

Comment:
Should this and the other be `sunset` instead of `deprecation`? I think @brightbyte was suggesting following the IETF draft in https://www.rfc-editor.org/rfc/rfc8594.html#section-3 (if `deprecation` is just a magical alias for `sunset`, please advise). No strong feeling here, just wanted to make sure I understood correctly!

Reply:
I think you are right, updating the patch.

Reply:
The Sunset header is appropriate when we have set a data for decomissioning. 
There is also a draft for a Deprecation header, which can be used earlier in the life cycle: https://greenbytes.de/tech/webdav/draft-ietf-httpapi-deprecation-header-latest.html

Reply:
This is the spec Ι confused with the one you mentioned before @brightbyte